.github/workflows/lint-test.yml

```
@@ -133,7 +133,7 @@ jobs:
         LIBCST_PARSER_TYPE: "native"
       run: |
         nox -s codemod -- run-all
-        if [ -n "$(git status --porcelain disnake/)" ]; then
+        if [ -n "$(git status --porcelain disnake/ tests/ test_bot/ scripts/ examples/ noxfile.py)" ]; then
```

```suggestion
        if [ -n "$(git status --porcelain)" ]; then
```
it shouldn't be a problem to check the entire repo

---

lets try it and see

---

seems to be good, it would have complained already if it wasn't.